dotnet-desktop-guide/framework/wpf/advanced/how-to-override-metadata-for-a-dependency-property.md

```diff
@@ -9,6 +9,7 @@ helpviewer_keywords:
   - "dependency properties [WPF], overriding metadata for"
   - "overriding metadata for dependency properties [WPF]"
 ms.assetid: f90f026e-60d8-428a-933d-edf0dba4441f
+description: Learn how to override default dependency property metadata from an inherited class by calling the OverrideMetadata method and providing type-specific metadata.
```

Comment:
For readability I suggest:
Learn how to override "the" default dependency property "of" metadata from an inherited class by calling the OverrideMetadata method and providing type-specific metadata.